크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,31 @@
+function solution(board, moves) {
+    const LENBOARD = board.length;
+    let newBoard = board.map((item) => []);
+    let pushedIcon = [];
```

Comment:
```suggestion
    const lenBoard= board.length;
    const newBoard = board.map((item) => []);
    const pushedIcon = [];
```
대문자 표기법에 대해 조금 알아봤는데 변수를 대문자로만 이룰 경우는 어떤일이 있어도 변하지 않는 값을 담을 때 대문자로 표기한다고 합니다. ex) 환경변수  저 경우에는 매개변수인 board가 어떤 값이 들어오냐에 따라 값이 변하기 떄문에 기존의 카멜케이스를 사용하여 변수를 사용하는게 맞고 참조타입의 값은 const로 선언해주세요